nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -1646,9 +1646,8 @@ impl<'a> BlueprintBuilder<'a> {
     /// ordinarily only come from RSS.
     ///
     /// TODO-cleanup: Remove when external DNS addresses are in the policy.
-    #[cfg(test)]
     #[track_caller]
-    pub fn add_external_dns_ip(&mut self, addr: IpAddr) {
+    pub(crate) fn add_external_dns_ip(&mut self, addr: IpAddr) {
         self.external_networking()
             .expect("failed to initialize external networking allocator")
```

Comment:
I haven't gotten far enough into the PR to see why this lost its `#[cfg(test)]`, but if this is usable in non-test code, I think we need to make it fallible instead of unwrapping here.

Reply:
Ah yeah, good catch. Thanks!

Reply:
Fixed.